timemachine/fe/utils.py

```diff
@@ -241,6 +241,13 @@ def sanitize_energies(full_us, lamb_idx, cutoff=10000):
     return np.where(abs_us < cutoff, full_us, np.inf)
 
 
+def read_sdf(fname: str) -> List[Chem.Mol]:
```

Comment:
nit: Would be nice if this was `Union[str, Path]`, as you already cast `fname` to str and use pathlib a fair amount

Reply:
Will change!

Reply:
Expanded type alias in https://github.com/proteneer/timemachine/pull/892/commits/fff1a54ca0e34a41cfa7c9a52604f7e02a6bf44a